recipes/ux.rb

```diff
@@ -8,12 +8,19 @@
 #
 
 log "Create jellyfish user"
-
 user "jellyfish" do
   comment "jellyfish user"
   shell "/bin/bash"
 end
 
+log "Create jellyfish sudoers file"
+file "/etc/sudoers.d/jellyfish" do
```

Comment:
What's the reason to add jellyfish to the sudoers file?  If we need this there is no point in running UX as jellyfish user.